myzappi-alexa-lambda/src/main/java/com/amcglynn/myzappi/graphing/EnergyUsageVisualisation.java

```diff
@@ -88,9 +88,9 @@ private LocalTime getLocalTime(ZappiHistory history, ZoneId userZone) {
     private byte[] generateEnergyUsageChartBytes(double[] imported, double[] exported, double[] consumed, ZappiDaySummary summary) {
         var graph = new MyEnergiGraph(dimension);
         if (summary != null) {
-            graph.addSeries(imported, summary.getImported() + "kWh", Color.RED);
-            graph.addSeries(exported, summary.getExported() + "kWh", Color.YELLOW);
-            graph.addSeries(consumed, summary.getConsumed() + "kWh", Color.GREEN);
+            graph.addSeries(imported, "Import: " + summary.getImported() + "kWh", Color.RED);
```

Comment:
This was a defect where if two series had the same name, the graphing library would fail. This would occur early in the morning when exported and consumed were both 0.0kWh. Needed to prefix both series with Import, Export and Consumed